src/overlay/TCPPeer.cpp

```diff
@@ -441,8 +441,11 @@ TCPPeer::startRead()
 
     mIncomingHeader.clear();
 
-    CLOG(DEBUG, "Overlay") << "TCPPeer::startRead " << mSocket->in_avail()
-                           << " from " << toString();
+    if (Logging::logDebug("Overlay"))
+    {
+        CLOG(DEBUG, "Overlay") << "TCPPeer::startRead " << mSocket->in_avail()
```

Comment:
we should just move to a proper logging library for which we don't need to do this all over the place

Reply:
Well, yes, but I'm going to start on that _next_ month. This is for the meantime.